### ISSUE TITLE
ncdump for opendap address failed (netcdf-c 4.7.0)

### ISSUE THREAD

**Issue by dkobashi** [Unidata/netcdf-c#1414]
> Please provide as much of the following information as you can, as applicable to the issue being reported.  Naturally, not all information is relevant to every issue, but the more information we have to start, the better!

### Environment Information

> Feel free to skip this if the issue is related to documentation, a feature request, or general discussion.

* What platform are you using? (please provide specific distribution/version in summary)
  * [ x ] Linux
  * [ ] Windows
  * [ ] OSX
  * [ ] Other
  * [ ] NA
* 32 and/or 64 bit?
  * [ ] 32-bit
  * [X ] 64-bit
* What build system are you using?
  * [X ] autotools (`configure`)
  * [ ] cmake
* Can you provide a sample netCDF file or `C` code to recreate the issue?
  * [X ] Yes (please attach to this issue, thank you!)
  * [ ] No
  * [ ] Not at this time

### Summary of Issue
I installed netCDF-4.7.0 with opendap support. When I tried ncdump with opendap url, I got "NetCDF: Unknown file format". 

output of nc-config is below.
```nc-config --all

 This netCDF 4.7.0 has been built with the following features: 

  --cc            -> gcc
  --cflags        -> -I/home/dj/software/netcdf/4.7.0-gnu/include -I/home/dj/software/curl/include -I/home/dj/software/netcdf/hdf5/include -I/home/dj/software/netcdf/zlib/include
  --libs          -> -L/home/dj/software/netcdf/4.7.0-gnu/lib -lnetcdf
  --static        -> -lhdf5_hl -lhdf5 -lm -ldl -lz -lcurl 

  --has-c++       -> no
  --cxx           -> 

  --has-c++4      -> no
  --cxx4          -> 

  --has-fortran   -> no
  --has-dap       -> yes
  --has-dap2      -> yes
  --has-dap4      -> yes
  --has-nc2       -> yes
  --has-nc4       -> yes
  --has-hdf5      -> yes
  --has-hdf4      -> no
  --has-logging   -> no
  --has-pnetcdf   -> no
  --has-szlib     -> no
  --has-cdf5      -> yes
  --has-parallel4 -> no
  --has-parallel  -> no

  --prefix        -> /home/dj/software/netcdf/4.7.0-gnu
  --includedir    -> /home/dj/software/netcdf/4.7.0-gnu/include
  --libdir        -> /home/dj/software/netcdf/4.7.0-gnu/lib
  --version       -> netCDF 4.7.0
```

### Steps to reproduce the behavior
Installation: 
```
LDFLAGS="-L/home/dj/software/curl/lib -L/home/dj/software/netcdf/zlib/lib -L/home/dj/software/netcdf/hdf5/lib" CPPFLAGS="-I/home/dj/software/curl/include -I/home/dj/software/netcdf/hdf5/include -I/home/dj/software/netcdf/zlib/include" ./configure --prefix=/home/dj/software/netcdf/4.7.0-gnu 

make check install
```
Then
```
ncdump -h https://tds.hycom.org/thredds/dodsC/GLBv0.08/expt_93.0
ncdump: https://tds.hycom.org/thredds/dodsC/GLBv0.08/expt_93.0: https://tds.hycom.org/thredds/dodsC/GLBv0.08/expt_93.0: NetCDF: Unknown file format
```

**Comment by DennisHeimbigner** [comment]
Ok, I can reproduce. I will investigate.

**Comment by DennisHeimbigner** [comment]
do the command
> ncdump
with no arguments.
At the end of the output, it will indicate the version.
Please report it to us.

**Comment by dkobashi** [comment]
netcdf library version 4.6.3 of Jun  2 2019 14:43:28 $.

Now I am confused.
``` 
which ncdump
/home/dj/software/netcdf/4.7.0-gnu/bin/ncdump
```
The version should be 4.7.0.

**Comment by DennisHeimbigner** [comment]
So am I.

**Comment by DennisHeimbigner** [comment]
I think you need to look in /usr/bin or /usr/local/bin to see if that is where
the ncdump 4.6.3 is located.

**Comment by dkobashi** [comment]
4.6.3 was the version installed before 4.7.0 without opendap because I had an issue on curl installed by default on my ubuntu when installing 4.6.3 (then I installed curl from source).

I downloaded the latest version of netcdf (4.7.1?) and installed with cmake and that worked fine. Then I removed version 4.6.3 and re-installed 4.7.0 with automake and that worked fine too.
So I don't know. maybe I messed up something. Anyway I close this issue. Many thanks for your help.
